### ISSUE TITLE
Can't enable JSB by clicking in the status bar

### ISSUE THREAD

**Issue by fbuchinger** [benmmurphy/jsdeminifier_xpi#3]
I can't enable JSB by clicking the "JSB" text field in the status bar. I'm using JSB 1.0.5 with Firefox 8/Firebug 1.8.4/Win7. Even reloading the page or clearing the browsercache doesn't help.

As a few users have reported this issue in the Mozilla Addon comments, I decided to open a github ticket for it.

**Comment by benmmurphy** [comment]
thanks for the bug report! 

i'm been waiting for somebody to raise it on github so i could get some more information :)

does the text toggle between 'JSB OFF' and 'JSB ON' ?

**Comment by fbuchinger** [comment]
thanks for the fast reply... no, the text doesn't toggle at all... it just says "JSB"

**Comment by benmmurphy** [comment]
I was able to reproduce the issue and find out which line it was crashing on. However, once I started mucking around with the source I wasn't able to reproduce the issue anymore even if I reverted all my changes. I think it might have been breaking because I was accessing the tabContainer property of gBrowser before the window.load event had been fired. I made changes so I would only access tabContainer after the load event has been fired.

I think I might have fixed it but I can't be sure :( I have a new version if you would like to test:

https://github.com/downloads/benmmurphy/jsdeminifier_xpi/jsdeminifier.xpi
